.github/workflows/ci.yml

```diff
@@ -24,6 +24,24 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  setup:
+    runs-on: ubuntu-latest
+
+    outputs:
+      examples-matrix: ${{ steps.examples-matrix.outputs.examples-matrix }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4.1.1
+
+      - name: Calculate `examples-matrix` output
+        id: examples-matrix
+        working-directory: ./examples
+        env:
+          JQ_FILTER: >-
+            "examples-matrix=" + (split("\n") | map(select(length > 0)) | tostring)
+        run: find . -type f -name 'pyproject.toml' -printf '%h\n' | cut -c 3- | sort -u | jq -cRrs '${{ env.JQ_FILTER }}' >> "$GITHUB_OUTPUT"
```

Comment:
not sure how you feel about this, but I'd prefer that the output was printed on screen here so it shows up in job logs. I know it just immediately becomes jobs in the left hand sidebar, but I always appreciate the debugability of knowing hey, there's an extra element I didn't want, obviously this job is going to fail (or vice versa)

<img width="1457" alt="github output" src="https://github.com/pangeacyber/pangea-python/assets/14140268/031335e7-2366-4989-9915-99a3fa6f1308">

Reply:
Sure, done.